spec/webmachine/adapters/rack_spec.rb

```diff
@@ -34,3 +35,23 @@
     end
   end
 end
+
+describe Webmachine::Adapters::Rack do
+  let(:app) do
+    Webmachine::Application.new.tap do |app|
```

Comment:
`tap` is [not necessary](https://github.com/seancribbs/webmachine-ruby/blob/master/lib/webmachine/application.rb#L48).